[2022-03-02 18:00:05,487] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.run_script 2022-03-02T12:00:00+00:00 [queued]>
[2022-03-02 18:00:05,498] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.run_script 2022-03-02T12:00:00+00:00 [queued]>
[2022-03-02 18:00:05,499] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-02 18:00:05,499] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-03-02 18:00:05,499] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-02 18:00:05,510] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): run_script> on 2022-03-02T12:00:00+00:00
[2022-03-02 18:00:05,515] {standard_task_runner.py:52} INFO - Started process 173278 to run task
[2022-03-02 18:00:05,521] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'run_script', '2022-03-02T12:00:00+00:00', '--job-id', '130', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmp3w38nr_9', '--error-file', '/tmp/tmpz5v8yu2l']
[2022-03-02 18:00:05,522] {standard_task_runner.py:77} INFO - Job 130: Subtask run_script
[2022-03-02 18:00:05,562] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.run_script 2022-03-02T12:00:00+00:00 [running]> on host 531edfb863fc
[2022-03-02 18:00:05,603] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=run_script
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T12:00:00+00:00
[2022-03-02 18:00:05,604] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-02 18:00:05,605] {bash.py:158} INFO - Running command: python /app/scripts/main.py
[2022-03-02 18:00:05,617] {bash.py:169} INFO - Output:
[2022-03-02 18:00:10,247] {bash.py:173} INFO - Italy: CO2 386, mix [('gas', 21915, '52'), ('unknown', 3771, '9'), ('wind', 3633, '9')]
[2022-03-02 18:00:10,247] {bash.py:173} INFO - Portugal: CO2 248, mix [('gas', 2871, '42'), ('wind', 1487, '22'), ('solar', 842, '12')]
[2022-03-02 18:00:10,247] {bash.py:173} INFO - Norway: CO2 27, mix [('hydro', 14968, '86'), ('wind', 1813, '10'), ('hydro discharge', 443, '3')]
[2022-03-02 18:00:10,248] {bash.py:173} INFO - Finland: CO2 161, mix [('nuclear', 3072, '32'), ('wind', 2148, '22'), ('hydro', 1847, '19')]
[2022-03-02 18:00:10,248] {bash.py:173} INFO - France: CO2 85, mix [('nuclear', 42527, '63'), ('hydro', 10399, '15'), ('gas', 8034, '12')]
[2022-03-02 18:00:10,248] {bash.py:173} INFO - Germany: CO2 466, mix [('coal', 24939, '42'), ('gas', 10197, '17'), ('solar', 6756, '11')]
[2022-03-02 18:00:10,316] {bash.py:177} INFO - Command exited with return code 0
[2022-03-02 18:00:10,339] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=run_script, execution_date=20220302T120000, start_date=20220302T180005, end_date=20220302T180010
[2022-03-02 18:00:10,361] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-02 18:00:10,398] {local_task_job.py:146} INFO - Task exited with return code 0
